docs/man/git-lfs-push.1.ronn

```diff
@@ -3,25 +3,28 @@ git-lfs-push(1) -- Push queued large files to the Git LFS endpoint
 
 ## SYNOPSIS
 
-`git lfs push` <remote> [branch]<br>
+`git lfs push` <remote> [ref]<br>
 `git lfs push` --object-id <remote> <oid1> <oid2> ...
 
 ## DESCRIPTION
 
-Upload Git LFS files to the configured endpoint for the current Git remote.
-
-This command shouldn't be necessary since Git LFS automatically sets up a
-pre-push hook for each repository.
+Upload Git LFS files to the configured endpoint for the current Git remote.  By
+default, it filters out objects that are already referenced by the local clone
+of the remote.
 
 ## OPTIONS
 
+* `--dry-run`:
+    Print the files that would be pushed, without actually pushing them.
+
+* `--all`:
+    This pushes all objects for the given ref or current branch, regardless of
+    whether they are referenced in the local clone of the remote.
```

Comment:
Should really mention the difference between providing a ref on the command line & not when `--all` is provided.